### PR TITLE
Extract non localised strings

### LINKED ISSUE #6492: Extract non-localised strings
There are still a lot of strings that have not been extracted to use the localisation mechanism.

The task here is to go through the UI and pull any strings out that have not been localised.

To aid doing this, you can press `shift + l` to toggle the locale between `None` and the current locale - this quickly shows you for a given page, which strings have not been localised.

Priority order:

- Home page
- Cluster Explorer (exploring a given cluster)
- Global Settings
- Users and Authentication
- Cluster Management
- Continuous Delivery

You should create a docs page with info for localising, for other developers to reference.

Things to come back to:

- Kubernetes resource states - shown in badges - need to re-work this to support 18n

### PR DESCRIPTION
**Pull request by bisht-richa**
### Summary
Fixes #6492

Fixed issues in some section pages.

Mostly badges, workspaces, names directly from Ids, links, and descriptions are not having localized strings.
Needs someone's help to fix I have updated those page details in the Zube card.

https://zube.io/rancher/dashboard-ui/c/6527

### To test
Press `shift+L`  to check if the contents have localized strings.

### CODE REVIEW

**Comment by nwmac** [comment]
@bisht-richa Can this be moved out of draft?

**Comment by catherineluse** [comment]
@bisht-richa Can you please rebase this on master?

**Review by richard-cox — CHANGES_REQUESTED** [review]
The PR diff doesn't look correct. This probably needs a rebase and manually checking each file. At first glance
- There's new files outside of `shell` 
- Some files haven't been merged correctly (for instance Carousel.vue)
- Some files show whitespace differences (for instance ExplorerProjectsNamespaces.vue)

I think it's worth going reviewing the confluence page on PRs (Development/Pull Requests). There's a good section on there that covers this sort of thing.

File: shell/edit/auth/azuread.vue [hunk 1]
```diff
@@ -184,6 +184,8 @@ export default {
       if ( oldEndpointKey ) {
         this.endpoint = this.determineEndpointKeyType(OLD_ENDPOINTS);
         this.oldEndpoint = true;
+      } else if ( newEndpointKey ) {
```
Comment (richard-cox) [inline shell/edit/auth/azuread.vue:187]: This does exactly the same as the else case

**Review by richard-cox — CHANGES_REQUESTED** [review]
There's still a lot of files changed that aren't to do with this issue
- .github/workflows/test.yaml
- shell/assets/images/generic-plugin.svg
- shell/components/Carousel.vue (this brings back a bug)
- shell/components/ExplorerProjectsNamespaces.vue (template)
- shell/components/Tabbed/index.vue
- shell/config/product/manager.js
- shell/edit/workload/index.vue
- shell/pages/c/_cluster/_product/projectsnamespaces.vue.
- store/prefs.js

Please make sure you go through the PRs `Files changed`

**Comment by bisht-richa** [comment]
Re open it

**Pull request closed without merging**